### PR TITLE
feat: cli: list claims and remove expired claims

### PR DESCRIPTION
**Pull request by geoff-vball**
## Related Issues
Closes https://github.com/filecoin-project/lotus/issues/9271

## Proposed Changes
Adds a cli command to list all claims for a given provider.
Adds a cli command to remove expired claims for a given provider.

## Sample Output
Screenshot 2023-01-12 at 12 43 23 PM

## Checklist

Before you mark the PR ready for review, please make sure that:

- [ ] Commits have a clear commit message.
- [ ] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: cli/filplus.go [hunk 1]
```diff
@@ -403,6 +490,99 @@ var filplusRemoveExpiredAllocationsCmd = &cli.Command{
...
+	Usage:     "remove expired claims (if no claims are specified all eligible claims are removed)",
+	ArgsUsage: "providerAddress Optional[...claimId]",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "from",
+			Usage: "optionally specify the account to send the message from",
+		},
+	},
+	Action: func(cctx *cli.Context) error {
+		if cctx.NArg() < 1 {
```
Comment (arajasek) [inline cli/filplus.go:504]:
```suggestion
		if !cctx.Args().Present() {
```

**Review by arajasek — APPROVED** [review]
Can we fill in the PR description, including sample output of the list command?

File: cli/filplus.go (hunk 1 above)
Comment (geoff-vball) [reply cli/filplus.go:504]: No I hate it

File: cli/filplus.go [hunk 2]
```diff
@@ -403,6 +490,99 @@ var filplusRemoveExpiredAllocationsCmd = &cli.Command{
 	},
 }
 
+var filplusRemoveExpiredClaimsCmd = &cli.Command{
+	Name:      "remove-expired-claims",
+	Usage:     "remove expired claims (if no claims are specified all eligible claims are removed)",
```
[the remark this answers is not in the document]
Comment (geoff-vball) [reply cli/filplus.go:495]: Yis

**Pull request merged**